tidb-cloud/connect-to-tidb-cluster.md

```diff
@@ -26,7 +26,7 @@ After your TiDB cluster is created on TiDB Cloud, you can use one of the followi
 
 To connect to a Serverless Tier cluster via standard connection, perform the following steps:
 
-1. Navigate to the **Clusters** page.
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page.
 
 2. Locate your cluster, and click **Connect** in the upper-right corner of the cluster area. A connection dialog box is displayed.
 
```

Comment:
L33: To keep it consistent with UI, use "**Connect with a SQL Client**"

<img width="1196" alt="image" src="https://user-images.githubusercontent.com/60599231/206976530-2ab06dc5-006b-4a1c-846e-08b39c55ad50.png">